mabwiser/neighbors.py

```diff
@@ -92,6 +97,18 @@ def _get_nhood_predictions(self, lp, indices, row_2d, is_predict):
         else:
             return lp.predict_expectations(row_2d)
 
+    def _get_no_nhood_predictions(self, lp, is_predict):
+
+        if is_predict:
+            # if no_nhood_prob_of_arm is None, select a random int
+            # else, select a non-uniform random arm
+            # choice returns an array, hence get zero index
+            rand_int = lp.rng.choice(len(self.arms), size=1, p=self.no_nhood_prob_of_arm)[0]
```

Comment:
FYI, with single item arrays you can also use `array.item()`. No need to change anything.

Reply:
Good to know!